src/commands/dev.ts

```diff
@@ -200,6 +200,57 @@ export async function command(commandOptions: CommandOptions) {
   const messageBus = new EventEmitter();
   const mountedDirectories: [string, string][] = [];
 
+  onProcessExit(() => {
+    hmrEngine.disconnectAllClients();
+  });
+
+  console.log = (...args) => {
+    messageBus.emit('CONSOLE', {level: 'log', args});
+  };
+  console.warn = (...args) => {
+    messageBus.emit('CONSOLE', {level: 'warn', args});
+  };
+  console.error = (...args) => {
+    messageBus.emit('CONSOLE', {level: 'error', args});
+  };
```

Comment:
These were moved up to the start of the `dev` command

Reply:
![Screen Shot 2020-06-12 at 12 46 06 PM](https://user-images.githubusercontent.com/1369770/84536280-b9369000-acaa-11ea-96c9-1dfe94251862.png)

Install errors still seem to work

Reply:
awesome!